rdkit/Chem/Features/ShowFeats.py

```diff
@@ -194,33 +203,24 @@ def ShowMolFeats(mol, factory, viewer, radius=0.5, confId=-1, showOnly=True, nam
     if useFeatDirs:
       ps = []
       if family == 'Aromatic':
-        ps, fType = FeatDirUtils.GetAromaticFeatVects(
+        ps, fType = FeatVectsDictMethod[(family, None)](
           mol.GetConformer(confId), feat.GetAtomIds(), pos, scale=1.0)
+      
       elif family == 'Donor':
         aids = feat.GetAtomIds()
         if len(aids) == 1:
           featAtom = mol.GetAtomWithIdx(aids[0])
           hvyNbrs = [x for x in featAtom.GetNeighbors() if x.GetAtomicNum() != 1]
-          if len(hvyNbrs) == 1:
-            ps, fType = FeatDirUtils.GetDonor1FeatVects(mol.GetConformer(confId), aids, scale=1.0)
-          elif len(hvyNbrs) == 2:
-            ps, fType = FeatDirUtils.GetDonor2FeatVects(mol.GetConformer(confId), aids, scale=1.0)
-          elif len(hvyNbrs) == 3:
-            ps, fType = FeatDirUtils.GetDonor3FeatVects(mol.GetConformer(confId), aids, scale=1.0)
+          ps, fType = FeatVectsDictMethod[(family, len(hvyNbrs))](
+            mol.GetConformer(confId), aids, scale=1.0)
```

Comment:
I suggest to make this two statements to make it easier to read.

```
method = FeatVect...
ps, fType = method(mol.Get...)
```

Reply:
OK, I got your idea! I would fix it in the next patch. Thank you for your review.